docusaurus/src/components/RequestERD/index.jsx

```diff
@@ -0,0 +1,63 @@
+import { useLocation } from "@docusaurus/router";
+import React, { useEffect, useState } from "react";
+import { EmailModal } from "./EmailModal";
+import styles from "./RequestERD.module.css";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+const { getRegistryEntry } = require("../../remark/utils");
+const { getFromPaths } = require("../../helpers/objects");
+
+export const RequestERD = () => {
```

Comment:
Let's rename this file to `RequestERD.jsx` to be consistent with other components in this repo